src/VisualStudio/Core/Test.Next/Remote/SerializationValidator.cs

```diff
@@ -220,8 +220,13 @@ internal async Task VerifyAssetAsync(DocumentStateChecksums documentObject)
 
         internal async Task VerifySolutionStateSerializationAsync(Solution solution, Checksum solutionChecksum)
```

Comment:
i would recommend looking at the product changes first before test changes.

Reply:
but, basically, the tests are updated now that we both make a request for a the solution-comp-state, and then the solution-state.